### DISCUSSION TITLE
Why there is a huge difference generated when I change the seed in mace parameters ?

### DISCUSSION THREAD

**Discussion by liubaoshuai1402** [ACEsuit/mace#1146] [Q&A] [answered]
Dear developers,
I try to train three mace models and use them to do active learning like here: Tutorial 2: MACE active learning and fine-tuning
However, when I use different seeds (123, 456, 789 respectively), the effect of training is very different.
I post three results,
for seed=123
<img width="3034" height="1834" alt="YSZH_MACE_model_run-123_train_Default_stage_two" src="https://github.com/user-attachments/assets/5e4a78b1-73c6-4d96-a0a3-d6a6c01b81de" />

for seed=456
<img width="3034" height="1834" alt="YSZH_MACE_model_run-456_train_Default_stage_two" src="https://github.com/user-attachments/assets/d327eb8f-c4e0-43aa-b375-eb327b236a8d" />

for seed=789
<img width="3034" height="1834" alt="YSZH_MACE_model_run-789_train_Default_stage_one" src="https://github.com/user-attachments/assets/8c89c2ed-9622-42b7-9395-753fa4722de9" />
Their final loss are about 0.046, 0.062, 0.052, respectively.
In the situation of seed=456, the loss even increased in the later stages.
**I want to know whether this difference is common or reasonable ?**
When I train the committee of models, could I use different epochs for different seed ?
After all, when seed=456, epoch=700 is obvious too large.
It seems that seed=123 have the best training effect and the default seed of mace is 123.
Is this a coincidence or are seed=123 you recommend after testing?
**Should I change seed=456 into other numbers ?** I think the model of seed=456 would not be suitable for being the committee

**Comment by ilyes319** [comment, marked as answer]
it is normal to see variations between seeds as mace is trained with a stochastic optimizer. Yes you can use different seeds for creating a committee. Note also that changing the seed in mace will also change the validation set if you use a random validation set such that your numbers are not exactly comparable. If you want to compare them, you need to provide a fixed validation set as an xyz file.
